packages/core/src/fhir-to-cda/cda-generators.ts

```diff
@@ -39,5 +39,16 @@ export function generateCdaFromFhirBundle(fhirBundle: Bundle): string {
     custodian,
     structuredBody
   );
-  return clinicalDocument;
+
+  const postProcessedXml = postProcessXml(clinicalDocument, oid);
+  return postProcessedXml;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function postProcessXml(xml: any, oid: string): string {
+  return xml
+    .replaceAll("<br>", "")
```

Comment:
```
<!DOCTYPE html>
<html>

<body>
    <p>
        Line start.<br />This is a new line.
    </p>
</body>

</html>
```

^ But this also works. Is there a difference? The reason I'm doing these replacements is because the CDA Validator does not allow `<br>` tags 

```{
                "element": "br",
                "error": "Character content is not allowed, because the content type is empty."
            },```

Reply:
Note I didn't say `<br/>` doesn't work, I know this is the correct form of a new line in HTML. I'm referring to HTML code that's placed inside `<text><content>...`.

Where did you get this message?

We can check files processed in the past to make sure the post-processing is not breaking stuff.

Reply:
Discussion taken to Slack and resovled